ffmpeg-shared.txt

```diff
@@ -1,11 +1,10 @@
-[Section]
-Name = FFmpeg-XP (shared library version)
-Version = 6.1-2009-bef151d
-Category = 2
-URLDownload = https://rwijnsma.home.xs4all.nl/files/ffmpeg/ffmpeg-6.1-2009-bef151d-win32-shared-xpmod-sse.7z
-SizeBytes = 27344813
-
-Description = A complete, cross-platform solution to record, convert and stream audio and video. Build by Reino.
-License = GPL
-SHA1 = 62b17a0eec6a21f1c0d55def22fa41816a27de43
-URLSite = https://forum.doom9.org/showthread.php?t=181802
+[Section]
+Name = FFmpeg-XP (shared library version)
+Version = 6.1-2009-bef151d
+License = GPL
+Description = A complete, cross-platform solution to record, convert and stream audio and video. Build by Reino.
+Category = 2
+URLSite = https://forum.doom9.org/showthread.php?t=181802
+URLDownload = https://rwijnsma.home.xs4all.nl/files/ffmpeg/ffmpeg-6.1-2009-bef151d-win32-shared-xpmod-sse.7z
+SHA1 = 62b17a0eec6a21f1c0d55def22fa41816a27de43
+SizeBytes = 27344813
```

Comment:
In this FFMPEG file (and some similar related ones) you changed only the encoding to fit our expectations?
Or are there any other intended changes in here?
Was it being "changed" as part of the PR intended at all?

Reply:
Yep, I changed it as the standard we said in Wiki (CR-LF and UTF-8 without BOM).
Discovered during the Git sync, because git autofix (CR-LF'ed) it.

Reply:
Thx. Resolved.